rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -1653,9 +1653,25 @@ def test_log_trimming(bucket, config):
         if len(output2) == 0:
             log.info(f"{config.log_trimming} log is empty after the interval")
         else:
-            raise TestExecError(
-                f"{config.log_trimming} log is not empty after the interval"
-            )
+            if config.log_trimming == "mdlog":
+                retry = 25
```

Comment:
can we reduce the retries to 20?

Reply:
@viduship , with 20 retry we observed still mdlog is not empty.
hence added retry of 25